src/main/resources/static/client.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html ng-app="pushServerClient">
+<head>
+    <meta charset="UTF-8">
+    <title>Client</title>
+
+    <style>
+        .liveScore {
+            color : blue;
+        }
+
+        li {
+            list-style: none;
+            padding: 0 0 10px 0;
+        }
+    </style>
+</head>
+<body>
+<div class="liveScore" ng-controller="pushServerController as ctrl">
+    <p>Notification list:</p>
+    <ul>
+        <li ng-repeat="message in ctrl.messages">
+            From: {{message[0].senderName}} <br/>
+            To: {{message[0].receiverName}} <br/>
+            Message: {{message[0].message}}
+        </li>
+    </ul>
+</div>
+
+<script src="sockjs-0.3.4.js"></script>
+<script src="stomp.js"></script>
+<script src="angular.js"></script>
+<script src="angular-stomp.js"></script>
+<script type="text/javascript">
+    angular
+        .module('pushServerClient', ['AngularStompDK'])
+        .config(stompConfig)
+        .controller('pushServerController', pushServerController);
+
+    function stompConfig(ngstompProvider)
+    {
+        console.log("Configurating connection...")
+        ngstompProvider
+            .url('http://localhost:8080/livescore-websocket')
+            .class(SockJS)
+            .autoConnect(true)
+            .reconnectAfter(10); // long time for testing purposes
+    }
+
+    function pushServerController(ngstomp, $http, $scope)
```

Comment:
`$http` is not used, can you make sure this is needed?

Reply:
Removed `$http`, it was unnecessary.